examples/teleport-usage/teleport_usage.py

```diff
@@ -0,0 +1,124 @@
+"""
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import json
+import datetime
+import boto3
+import os
+
+from boto3.dynamodb.conditions import Key, Attr
+
+
+# Check for AWS DynamoDB table name and region ENV, if none prompt
+table_name = os.getenv("TABLE_NAME")
+if not TABLE_NAME:

```

Comment:
This isn't going to work, right?

```suggestion
if not table_name:
```